interfaces/builtin/spi.go

```diff
@@ -55,7 +55,7 @@ func (iface *spiInterface) StaticInfo() interfaces.StaticInfo {
 	}
 }
 
-var spiDevPattern = regexp.MustCompile("^/dev/spidev[0-9].[0-9]+$")
+var spiDevPattern = regexp.MustCompile(`^/dev/spidev[0-9]\+.[0-9]+$`)
```

Comment:
Did you mean `^/dev/spidev[0-9]+.[0-9]+$` ?

Reply:
@bboozzoo yeah mistake, should be 
`^/dev/spidev[0-9]+\.[0-9]+$`